src/settingsKeyMapping.js

```diff
@@ -210,6 +210,10 @@ const settingsKeyMapping = {
             FIXED: 'fixed',
         },
     },
+    keyIncludeZeroValuesInAnalytics: {
+        label: 'key_skip_zero_values_in_analytics_table_export',
```

Comment:
The key is now "Include" zeros rather than "Skip" zeros, so the localisation key and string (in .properties file) need to be updated accordingly too.

Reply:
My bad! Updated and removed old translation